xbmc/utils/StringUtils.cpp

```diff
@@ -759,6 +759,61 @@ std::vector<std::string> StringUtils::Split(const std::string& input, const char
   return results;
 }
 
+std::vector<std::string> StringUtils::Split(const std::vector<std::string> &input, const std::vector<std::string> &delimiters, unsigned int iMaxStrings /* = 0 */)
```

Comment:
Not sure. SplitMulti had to have a different name to differentiate between split with a string that should be applied as a single delimter, and a string where each character is a deliimter. SplitMulti is also unused now, I wrote it to do the initial version of tag repair, so do I remove it? Or do I replace it?